apps/full-site-editing/package.json

```diff
@@ -0,0 +1,41 @@
+{
+	"name": "@automattic/full-site-editing",
+	"version": "1.0.0-alpha.0",
+	"description": "Plugin and theme enabling full site editing with the block editor.",
+	"sideEffects": true,
+	"repository": {
+		"type": "git",
+		"url": "git://github.com/Automattic/wp-calypso.git",
+		"directory": "apps/full-site-editing"
+	},
+	"private": true,
+	"author": "Automattic, Inc.",
+	"license": "GPL-2.0-or-later",
+	"bugs": {
+		"url": "https://github.com/Automattic/wp-calypso/issues"
+	},
+	"homepage": "https://github.com/Automattic/wp-calypso",
+	"scripts": {
+		"plugin": "webpack --folder='plugin' --env.WP='true'",
+		"dev:plugin": "npx lerna run plugin",
```

Comment:
Does `npm run plugin` produces a different result? We don't use `npm lerna run` on the existing `o2-blocks` and `wpcom-block-editor` apps, but maybe you found some issues because you ran the commands inside the `full-site-editing` dir?

Reply:
Right, I'm almost sure it gave me some issues, but I've done a lot of changes on these scripts and tbh they might have been caused by something else.
I'll try converting this with the whole `npx lerna run` from the root, and see what happens.